### ISSUE TITLE
Behavior of backspace inconsistent with other commands

### ISSUE THREAD

**Issue by vks** [jmacdonald/amp#55]
Backspace leaves you in insert mode, which is inconsistent with the other delete commands and undocumented.

**Comment by jmacdonald** [comment]
Right. I added that change recently after realizing that I was _very_ rarely using backspace without inserting another value. Agreed that it's inconsistent, though; you _can_ change that in the keymap if you'd like:

```yaml
keymap:
  normal:
    backspace:
      - buffer::backspace
      - application::switch_to_normal_mode
```

I've realized that this "switch to insert mode" behaviour is in the `buffer::backspace` command itself; I may move that into the default keymap instead (that'll make the above keymap only require `backspace: buffer::backspace`). Leaving this issue open as a reminder to do that; I'll follow up here once that's landed.
